user_docs/en/userGuide.md

```diff
@@ -2243,6 +2243,36 @@ Disabling this option may improve readability.
 
 To toggle show selection from anywhere, please assign a custom gesture using the [Input Gestures dialog](#InputGestures).
 
+##### Formatting display {#BrailleSettingsFormattingDisplay}
```

Comment:
can you fix up this section to make sure sentences are on new lines, and LibLouis is cased correctly.

Reply:
Ditto